Bug: Grey particles when breaking grass.

![image](https://user-images.githubusercontent.com/61436375/89710233-f51f6600-d978-11ea-95f3-1380c8533f66.png)

**expected result:**
When the block breaks, the correct particles should show (like in singleplayer, green)

**What actually happens:**
Grey particles (that appear to have the wrong texture) show

**How to reproduce this bug:**
- Break a block

Hi @HerryYT. Currently I am following the readme to install but I am hitting this issue when I do `npm run build`

```bash
src/prismarine.ts:70:53 - error TS2345: Argument of type 'World | null' is not assignable to parameter of type 'World'.
  Type 'null' is not assignable to type 'World'.

70                     connection, connection.address, world, this
                                                       ~~~~~
Found 1 error.
```

Here is my version of nodejs and npm
nodejs - v14.14.0
npm - 6.14.8

--- a/src/network/handler/player-action-handler.js
+++ b/src/network/handler/player-action-handler.js
@@ -6,6 +6,7 @@ const Prismarine = require('../../Prismarine');
 const WorldEventPacket = require('../packet/world-event');
 const LevelEventType = require('../type/level-event-type');
 const Vector3 = require('../../math/vector3').default;
+const UpdateBlockPacket = require('../packet/update-block');
 
 
 class PlayerActionHandler {
@@ -60,6 +61,18 @@ class PlayerActionHandler {
                     let chunk = await player.getWorld().getChunkAt(
                         blockVector3.getX(), blockVector3.getZ()
                     );
+                    let blockId = chunk.getSubChunk(blockVector3.getY()).getBlockId(blockVector3.getX(), blockVector3.getY(), blockVector3.getZ());
+
+                    pk = new UpdateBlockPacket();
+                    pk.x = blockVector3.getX();
+                    pk.y = blockVector3.getY();
+                    pk.z = blockVector3.getZ();
+                    pk.BlockRuntimeId = blockId;
+
+                    for (let onlinePlayer of server.getOnlinePlayers()) {
+                        onlinePlayer.sendDataPacket(pk);
+                    }
+
                     chunk.setBlock(
                         blockVector3.getX() % 16, blockVector3.getY(), blockVector3.getZ() % 16, server.getBlockManager().getBlock('minecraft:air')
                     );

--- a/src/network/packet/update-block.js
+++ b/src/network/packet/update-block.js
@@ -1,0 +1,33 @@
+const DataPacket = require('./packet');
+const Identifiers = require('../identifiers');
+
+
+class UpdateBlockPacket extends DataPacket {
+    static NetID = Identifiers.UpdateBlockPacket
+
+    /** @type {number} */
+    x = 0.0
+    /** @type {number} */
+    y = 0.0
+    /** @type {number} */
+    z = 0.0
+    /** @type {number} */
+    BlockRuntimeId
+    /** @type {number} */
+    Flags = 2
+    /** @type {number} */
+    Layer = 1
+
+    encodePayload() {
+        this.writeLFloat(this.x);
+        this.writeLFloat(this.y);
+        this.writeLFloat(this.z);
+
+        this.writeVarInt(this.BlockRuntimeId);
+
+        this.writeVarInt(this.Flags);
+
+        this.writeVarInt(this.Layer);
+    }
+}
+module.exports = UpdateBlockPacket;

